src/Microsoft.DotNet.Compiler.Common/LibraryExporterExtensions.cs

```diff
@@ -27,7 +28,21 @@ public static void CopyTo(this IEnumerable<LibraryAsset> assets, string destinat
 
             foreach (var asset in assets)
             {
-                File.Copy(asset.ResolvedPath, Path.Combine(destinationPath, Path.GetFileName(asset.ResolvedPath)), overwrite: true);
+                var file = Path.Combine(destinationPath, Path.GetFileName(asset.ResolvedPath));
+                File.Copy(asset.ResolvedPath, file, overwrite: true);
+                RemoveFileAttribute(file, FileAttributes.ReadOnly);
+            }
+        }
+
+        private static void RemoveFileAttribute(String file, FileAttributes attribute)
```

Comment:
Can this method be refactored with https://github.com/dotnet/cli/blob/rel/1.0.0/src/Microsoft.DotNet.Files/ContentFiles.cs#L87-L97 so there isn't duplication of code?